src/utils/functions.js

```diff
@@ -12,14 +15,75 @@ module.exports = {
 	switch (response.error) {
-	case "AuthenticationError": {
-		switch (response.description) {
-		case "API key is wrong":
-			return msg.channel.send("Error: API key has been set incorrectly.")
+		case "AuthenticationError": {
+			switch (response.description) {
+				case "API key is wrong":
+					return msg.channel.send("Error: API key has been set incorrectly.")
+			}
+			break
 		}
-		break
+		default:
+			msg.channel.send(`Error \`${response.error}\`: \`${response.description}\``)
+	}
+}
+
+/**
+ * 
+ * @param {Array} fields 
+ * @param {object} embedMsgOptions
+ * @param {embedMsgOptions.author} 
+ * @param {object} options
+ * @param {options.maxPageCount} maxPageCount - maximum number of things on the page
+ */
+async function createPagedEmbed(fields, embedMsgOptions, message, options = {}) {
+	if (!options.maxPageCount) options.maxPageCount = 25
+	if (options.maxPageCount > 25) options.maxPageCount = 25 // discord limit is 25 things per embed
+	let embed = new MessageEmbed(embedMsgOptions)
+	let page = 0
+	// if the amount of pages is 1 then there is actually only one page, f.e. 5/5 = 1 but the program will work with 0, 1
+	let maxPages = Math.floor(fields.length / options.maxPageCount)
+	if (maxPages === 1) maxPages = 0
+	embed.fields = fields.slice(0, options.maxPageCount)
+	let embedMsg = await message.channel.send(embed)
+
+	const setData = async () => {
+		const start = page * options.maxPageCount
+		embed.fields = fields.slice(start, start + options.maxPageCount)
+		embedMsg = await embedMsg.edit(null, embed)
+	}
+	const removeReaction = async (emoteName) => {
+		embedMsg.reactions.cache.find(r => r.emoji.name === emoteName).users.remove(message.author.id)
 	}
-	default:
-		msg.channel.send(`Error \`${response.error}\`: \`${response.description}\``)
+	// if there is only 1 page then no sense to make arrows, just slows other reactions down
+	if (maxPages) {
+		embedMsg.react("⬅️")
+		embedMsg.react("➡️")
 	}
+	embedMsg.react("🗑️")
+
+
+	const backwardsFilter = (reaction, user) => reaction.emoji.name === "⬅️" && user.id === message.author.id
+	const forwardsFilter = (reaction, user) => reaction.emoji.name === "➡️" && user.id === message.author.id
+	const removeFilter = (reaction, user) => reaction.emoji.name === "🗑️" && user.id === message.author.id
+
+	const backwards = embedMsg.createReactionCollector(backwardsFilter, { timer: 120000 })
+	const forwards = embedMsg.createReactionCollector(forwardsFilter, { timer: 120000 })
+	const remove = embedMsg.createReactionCollector(removeFilter, { timer: 120000 })
+
+	backwards.on("collect", (reaction) => {
+		page--
+		removeReaction("⬅️") // remove the user's reaction no matter what
+		if (page == -1) page = 0
+		else setData()
+	})
+	forwards.on("collect", (reaction) => {
+		page++
+		removeReaction("➡️") // remove the user's reaction no matter what
+		if (page > maxPages) page = maxPages
+		else setData()
+	})
+
+	remove.on("collect", () => {
+		embedMsg.delete()
+	})
```

Comment:
Im not sure if this is done automaticlly, but clean up needs to be considered, for example removing event connections.